src/index.js

```diff
@@ -14,7 +19,11 @@ import injectJSXWrapper from "./helpers/runtime/jsx-wrapper";
 
 export default function ({ Plugin, types: t }) {
   return new Plugin("incremental-dom", { visitor : {
-    Program: setupInjector,
+    Program: function(program, parent, scope, file) {
+      const options = get(file, ['opts', 'extra', 'incremental-dom']);
+      setPrefix(options);
+      setupInjector(program, parent, scope, file);
```

Comment:
I [tried](https://github.com/babel-plugins/babel-plugin-incremental-dom/pull/19#discussion_r40519491) to remove `t` from all the function calls. :stuck_out_tongue: